packages/agents/cartographer/poller/src/lib/operations/messagestatus.ts

```diff
@@ -12,18 +12,28 @@ export const updateMessageStatus = async () => {
 
   for (const domain of domains) {
     const limit = 100;
```

Comment:
We should be able to pull more than 100 from the DB, right? What would be a good value to use here that is okay on mem usage but still pulls in enough records? 1000?

Reply:
Yes, we could use larges pages. Updated it to 1000.